src/utilities/getAppBaseUrl.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { getAppGuiPath } from './getAppGuiPath'
```

Comment:
shouldn't this be `getAppBaseUrl`?

Reply:
Oh, I copied the wrong tests while splitting up the files 🤦🏻

Reply:
Fixed.